src/commands/run-tests.yml

```diff
@@ -1,105 +1,113 @@
+      folders using a colon-separated or semicolon-separated list.
     type: string
     default: ''
-  code-coverage-cobertura:
+  select-by-folder:
     description: >
-      Path to write code coverage report in Cobertura XML format.
+      Location of the folder used to select test suite elements, relative to the project root folder.
+      To create a test suite, MATLAB uses only the tests in the specified folder and its subfolders.
+      You can specify multiple folders using a colon-separated or semicolon-separated list.
     type: string
     default: ''
-  code-coverage-html:
+  select-by-tag:
     description: >
-      Path to write code coverage report in HTML format.
+      Test tag used to select test suite elements. To create a test suite, MATLAB uses only the test elements
+      with the specified tag.
     type: string
     default: ''
-  source-folder:
+  strict:
     description: >
-      Location of the folder containing source code, relative to the project root folder. The
-      specified folder and its subfolders are added to the top of the MATLAB search path. To
-      generate a code coverage report, MATLAB uses only the source code in the specified folder and
-      its subfolders. You can specify multiple folders using a colon-separated or a
-      semicolon-separated list.
-    type: string
-    default: ''
-  select-by-folder:
+      Option to apply strict checks when running tests, specified as `false` or `true`. If you specify
+      a value of `true`, the command generates a qualification failure whenever a test issues a warning.
+    type: boolean
+    default: false
+  use-parallel:
     description: >
-      Location of the folder used to select test suite elements, relative to the project root folder.
-      To generate a test suite, MATLAB uses only the tests in the specified folder and its subfolders.
-      You can specify multiple folders using a colon-separated or a semicolon-separated list.
+      Option to run tests in parallel, specified as `false` or `true`. If the test runner configuration
+      is suited for parallelization, you can specify a value of `true` to run tests in parallel. This
+      input requires a Parallel Computing Toolbox license.
+    type: boolean
+    default: false
+  output-detail:
+    description: >
+      Amount of event detail displayed for the test run, specified as `none`, `terse`, `concise`, `detailed`,
+      or `verbose`. By default, the command displays failing and logged events at the `detailed` level and test
+      run progress at the `concise` level.
     type: string
     default: ''
-  select-by-tag:
+  logging-level:
     description: >
-      Test tag used to select test suite elements. To generate a test suite, MATLAB uses only the test elements
-      with the specified tag.
+      Maximum verbosity level for logged diagnostics included for the test run, specified as `none`, `terse`,
```

Comment:
On the orb page, I see this values are quoted: 'None', ...
Can the user specify none instead of 'none'?

Reply:
> On the orb page, I see this values are quoted: 'None', ... Can the user specify none instead of 'none'?

I don't quite understand the question. Is the question about case sensitivity, wrapping the value in a string, or providing no value at all?

Reply:
Ok, in this case, let's keep it as is.